### PR TITLE
Add team Shield and Policy API labels to policies

### PR DESCRIPTION
**Pull request by fhielpos**

### CODE REVIEW

**Review by ssyno — APPROVED** [review]

**Review by stone-z — COMMENTED** [review]
from the slack thread, I think `managed` can be dropped since we won't use it now. Team label is :+1:

**Comment by fhielpos** [comment]
Done 😄

File: helm/kyverno-policies/values.yaml [hunk 1]
```diff
@@ -1,2 +1,5 @@
 kyverno-policies:
   podSecurityStandard: restricted
+  customLabels:
+    application.giantswarm.io/team: shield
+    policy.giantswarm.io/giantswarm-exempt: "false"
```
Comment (stone-z) [inline helm/kyverno-policies/values.yaml:5]: This is only needed when a policy doesn't have a team label :smile:

Comment (fhielpos) [reply helm/kyverno-policies/values.yaml:5]: Ah, ack, then I ditch it.

**Review by stone-z — APPROVED** [review]

**Pull request merged**